spec/presenters/summary/sections/justification_for_legal_aid_spec.rb

```diff
@@ -24,17 +24,17 @@
   end
 
   describe '#show?' do
-    context 'when there is no ioj' do
-      let(:ioj) { nil }
-
+    context 'when there is an ioj' do
       it 'shows this section' do
-        expect(subject.show?).to be(false)
+        expect(subject.show?).to be(true)
```

Comment:
Just reordering to be constant with this suite of tests (and renamed to be correct)